titus_isolate/isolate/workload_manager.py

```diff
@@ -161,6 +164,14 @@ def get_workloads(self):
     def get_isolated_workload_ids(self):
         return self.__cgroup_manager.get_isolated_workload_ids()
 
+    def is_isolated(self, workload_id):
+        noop_allocators = [NoopCpuAllocator, NoopResetCpuAllocator]
+        for allocator in noop_allocators:
```

Comment:
I understand why you're doing this, but the semantics `is_isolated` is a bit weird here

Reply:
Yeah, the name isn't great.  `is_handled`, `is_processed`?  On the other hand the workload is as _isolated_ as it's going to get.

Reply:
We can always revisit this if it becomes an issue.  I'm going to merge as its blocking dependent work.